docs/API_SPECS_SIMULATIONS.md

```diff
@@ -115,6 +115,10 @@ Content-Type: application/json; charset=utf-8
     {
       "Id": "engine-01",
       "Count": 1
+    },
+    {
+      "Id": "cab-01-proto",
+      "Count": 1
```

Comment:
the JSON snippets in this files are only examples, so I don't think there's need for any change here